build/ninja_main.py

```diff
@@ -129,16 +130,23 @@ def ShellFunctions(cc_sources, f, argv0):
 main() {
   ### Compile oils-for-unix into _bin/$compiler-$variant-sh/ (not with ninja)
 
-  local compiler=${1:-cxx}   # default is system compiler
-  local variant=${2:-opt}    # default is optimized build
-  local skip_rebuild=${3:-}  # if the output exists, skip build'
+  local compiler=${1:-cxx}        # default is system compiler
+  local variant=${2:-opt}         # default is optimized build
+  local translator=${3:-mycpp}    # default is the translator w/o optimizations
```

Comment:
the Usage string at the top of the generated file should be updated with the TRANSLATOR? argument

and note that it can be mycpp or mycpp-souffle

It seems like this will also help if we ever do a migration off MyPy